src/blocks/body.css

```
@@ -0,0 +1,6 @@
+body {
```

```suggestion
.body {
```
Краще стилізувати body по класу а не по тегу

---

fixed
UPD